ui/src/auth_container.jsx

```diff
@@ -17,117 +25,141 @@ export default React.createClass({
 
   getDefaultProps() {
     return {
-      localStorageKey: 'threeflows_auth_container'
+      localStorageKey: 'threeflow_email_registration'
```

Comment:
This probably doesn't matter but maybe for consistency and so that it doesn't cause problems later, can we add the 's' to the 'threeflow' in the `localStorageKey` so that it will be `threeflows_email_registration` instead of `threeflow_email_registration`.